test/unit/lib/utils/gpDataMapper.js

```diff
@@ -4,23 +4,31 @@ const gpDataMapper = require('../../../../app/lib/utils/gpDataMapper');
 const expect = chai.expect;
 
 describe('GP Surgery Data Mapper', () => {
-  it('for a GP Surgery that has POMI data and a url will return the url', () => {
-    const gpData = { bookingSystem: { supplier: 'EMIS', bookOnlineLink: 'http://example.com' } };
+  it('for a GP Surgery that has appointments POMI data and a url will return the url', () => {
+    const gpData = { onlineServices: { appointments: { supplier: 'EMIS', url: 'http://example.com' } } };
     const expectedResult = 'http://example.com';
     const result = gpDataMapper(gpData);
 
     expect(result).to.be.equal(expectedResult);
```

Comment:
I don't think so. It makes it more difficult to see the separation between Arrange, Act and Assert as it effectively merges Act and Assert into a single step.